src/styles/globals.css

```diff
@@ -0,0 +1,34 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+
+@font-face {
+    font-family: 'montserrat';
+    src: url("../assets/fonts/Montserrat.ttf") format('truetype')
+  }
+  
+  @font-face {
+    font-family: 'gabarito';
+    src: url('../assets/fonts/Gabarito.ttf') format('truetype')
+  }
+
+@layer {
+    /* Global text styles */
+    h1, h2, h3{
```

Comment:
Hi Nataly

I have reviewed the website and with this change, the color of the text "Bienvenido a Ecuador In Tech" in the Hero section seems to be dark :(

Reply:
Hi @IvanM9 ,
Thank you! Bug is fixed now :)